### ISSUE TITLE
Powered Cart rebalance \ return to game.

### ISSUE THREAD

**Issue by elfl0rd** [StrangeLoopGames/EcoIssues#11291]
In current state, powered cart is useless :(
At the technical level where it will be available to you (internal combustion engine) you really choose the truck.
I think that the recipe does not require mechanics workbench (only the work of a blacksmith, and basic engineer and ofc, not combustion engine - something like "low-power steam engine") would be very good to fill the gap between the carts and a steam truck.

Current state
wheelbarrow+small wood cart
-steam truck
--truck+powered cart

Wanted state
wheelbarrow+small wood cart(no profession)
-powered cart(blacksmith+basic engineer)
--steam truck(mechanics)
---truck(industry)

p.s.
Why am I against removing the powered cart from the game.
1) we have too big gap between big carts and steam truck (well seriously, you can make metal, brick and glass things, in 0.8 at this stage you already need big transports, but you're still hauling carts.)
2) I love powered cart model, and browse from the driver's seat

**Comment by elfl0rd** [comment]
Current state
wheelbarrow+small wood cart -> steam truck -> truck+powered cart

Wanted state
wheelbarrow+small wood cart(no profession) -> powered cart(blacksmith+basic engineer) -> steam truck(mechanics) -> truck(industry)

**Comment by elfl0rd** [comment]
Also... More vechicles - more resources consumption and more pollute :P

**Comment by Conchipedia** [comment]
Powered Cart Needs better steering, the original method has a very large turning circle so this led to people not wanting to craft it as it was very cumbersome, redesigning it so that it uses the **new and improved**  control would hopefully keep this very nice looking vehicle in the game.

**Comment by metachronism** [comment]
powered cart is intended to be useless.  its been replaced with updated art in the form of the steam truck

**Comment by elfl0rd** [comment]
um... closing and assign? lol.

**Comment by elfl0rd** [comment]
![default](https://user-images.githubusercontent.com/4980243/52747950-27573e00-2ff6-11e9-9b7e-4dd9541e5ed4.png)

**Comment by Conchipedia** [comment]
i want to try and keep it in the game, just maybe change it somehow to steer better so its not classed as useless, we have to keep some Original stuff in ECO ❤️

**Comment by keegan-o** [comment]
It would probably serve best as a free asset to give people as part of a 'Vehicle Modding Guide' or something, I think. It's not a priority to re-work the thing to put it back in the game, and it should have been removed from the tech tree long ago. We definitely have more vehicles we would like to add and I don't think the powered cart should take priority over them. Perhaps we'll find a place for it, but first we should be happy with all the other vehicles.

**Comment by elfl0rd** [comment]
My little rework

[PoweredCart.cs.txt](https://github.com/StrangeLoopGames/EcoIssues/files/2884575/PoweredCart.cs.txt)
Requie basic engineering 3 and
[LowPressureSteamEngine.cs.txt](https://github.com/StrangeLoopGames/EcoIssues/files/2884574/LowPressureSteamEngine.cs.txt)
Requie Smelting 5 and 10 ironpipe, 4 copper pipe and 30 ironingot(SmeltingSkill.MultiplicativeStrategy)

(Also need icon for LowPressureSteamEngine, but icon from CompactSteamEngine is appropriate)

**Comment by keegan-o** [comment]
That's a possibility, I'm sure that would work as far as the tech tree goes-- art wise though we would need to remove the giant covered wagon part of it I think and replace that with a stockpile visualizer like the others. Might still feel out of place even after that, but it's something we can try at some point.

**Comment by elfl0rd** [comment]
In fact, many people find that it looks good with a covered car.
For me - two issue, 1) position in tech tree, 2) control (steering)

**Comment by SeanAtr0n** [comment]
the design is inherently bad as far as vehicle controls go, imo we should get rid of it

**Comment by Conchipedia** [comment]
as in it  has 3 wheels unlike the new vehicles with 4?

**Comment by SeanAtr0n** [comment]
yup, 3 wheels with a high turn radius = extremely unstable vehicle

**Comment by D3nnis3n** [comment]
Actually, from experience - the car is neither used nor would be used in the state it is, no matter what we do. Driving it is just a pain, it's too big, too slow, has too high curving radius. Should just be removed, replacing it with the truck was the right thing to do.

**Comment by elfl0rd** [comment]
> Actually, from experience - the car is neither used nor would be used in the state it is, no matter what we do. Driving it is just a pain, it's too big, too slow, has too high curving radius. Should just be removed, replacing it with the truck was the right thing to do. To make it usable we actually would need to make it a steam truck, which we already did. Agree with Sean.

In fact, as my experience has shown, on low rates it is very even used.
(Ofc, with rebalance. You can see my "mod" above.)
This is also due to the fact that it has more slots.
Yes, it is uncomfortable to drive...
But the first machines were really like that!
In addition, its rebalance allows you to close the gap between the carts and the truck.

**Comment by D3nnis3n** [comment]
Quite some time since this discussion was active, but i actually still have the same opinion, therefore i move it over to suggestions. I feel we rather want to use the slot between carts and steam trucks with animal-powered carts in conjunction with animal husbandry. The powered cart is used by an extreme minority since the steam truck is in and despite it being cheaper, it's being skipped for good reason.
